pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraProduction.scala

```diff
@@ -165,6 +165,14 @@ trait SierraProduction {
       case _ => false
     }
 
+  private def marc264IsOnlyPunctuation(
```

Comment:
I wonder a bit about the name, the test nicely shows this matches `:` `,` or `blank` but the name suggests it won't match blank and will match other punctuation.  Maybe something more generic?

Reply:
Could split it into `isEmpty`, and `isPunctuation` to be more explicit.